README.md

```diff
@@ -1584,6 +1587,63 @@ Server: spray-can/1.3.3
 }
 ```
 
+## POST /api/workflows/:version/query
+
+This endpoint allows for querying workflows based on the same criteria
+as [GET /api/workflows/:version/query](#get-apiworkflowsversionquery).
+
+Instead of specifying query parameters in the URL, the parameters
+must be sent via the POST body. The request content type must be
+`application/json`. The json should be a list of objects. Each json
+object should contain a different criteria.
+
+cURL:
+
+```
+$ curl -X POST --header "Content-Type: application/json" -d "[{\"start\": \"2015-11-01\"}, {\"end\": \"2015-11-03\"}, {\"status\": \"Failed\"}, {\"status\": \"Submitted\"}]" "http://localhost:8000/api/workflows/v1/query"
```

Comment:
I don't know if it's specified somewhere else, but it could be useful to document that the filters are `OR`ed together